Before: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
After: 024467d03e7db9ef40534ffaec1a1fe2233dfb1248e12692445be914b5f514af
Patch: 1ecc2a48aefe9d66278e899f3a0a17980ca2830c87bbe2f4de64e7d856c9a9a0
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ Rakefile	2012-08-09 11:58:02.612251978 +1000
@@ -0,0 +1,6 @@
+raise "Rake must not be run directly. Either run via Bundler (bundle exec rake) or via bin/rake." unless defined?(Bundler)
+APP_ROOT ||= File.expand_path(File.dirname(__FILE__))
+require 'corvid/rake/tasks'
+
+# Set default task to test
+task :default => :test
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ corvid-features/corvid.rb	2012-08-09 12:32:01.750027917 +1000
@@ -0,0 +1,6 @@
+since_ver 1
+
+install {
+  copy_file       'Rakefile'
+  empty_directory 'lib'
+}
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ corvid-features/test_spec.rb	2012-08-09 12:28:47.208969779 +1000
@@ -0,0 +1,6 @@
+since_ver 1
+
+install {
+  copy_file               'test/bootstrap/spec.rb'
+  empty_directory         'test/spec'
+}
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ corvid-features/test_unit.rb	2012-08-09 12:28:47.228969888 +1000
@@ -0,0 +1,6 @@
+since_ver 1
+
+install {
+  copy_file               'test/bootstrap/unit.rb'
+  empty_directory         'test/unit'
+}
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ test/bootstrap/all.rb	2012-08-09 11:58:02.615585330 +1000
@@ -0,0 +1,11 @@
+# encoding: utf-8
+APP_ROOT = File.expand_path('../../..',__FILE__)
+require 'rubygems'
+require 'bundler/setup'
+require 'corvid/test/bootstrap/all'
+
+Bundler.require :default
+
+# Load test helpers
+Dir.glob("#{APP_ROOT}/test/helpers/**/*.rb") {|f| require f}
+
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ test/bootstrap/spec.rb	2012-08-09 11:58:02.615585330 +1000
@@ -0,0 +1,5 @@
+# encoding: utf-8
+$coverage_name = 'specs'
+require_relative 'all'
+require 'corvid/test/bootstrap/spec'
+
--- /dev/null	2012-07-30 09:37:23.066384672 +1000
+++ test/bootstrap/unit.rb	2012-08-09 11:58:02.615585330 +1000
@@ -0,0 +1,4 @@
+# encoding: utf-8
+$coverage_name = 'unit tests'
+require_relative 'all'
+require 'corvid/test/bootstrap/unit'
